source/_integrations/plugwise.markdown

```diff
@@ -4,7 +4,7 @@ description: "Plugwise Climate integration."
 logo: plugwise.png
 ha_category: Climate
 ha_iot_class: Local Polling
-ha_release: 0.98
+ha_release: 0.102
```

Comment:
Why should you change the release version? Please revert his

Reply:
Revert done